helpdesk_portal_priority/controllers/main.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+import odoo.http as http
+
+from odoo.addons.helpdesk_mgmt.controllers.main import HelpdeskTicketController
+
+_logger = logging.getLogger(__name__)
+
+
+class ApslHelpdeskTicketController(HelpdeskTicketController):
```

Comment:
This shouldn't be ApslHelpdeskTicketController
```suggestion
class HelpdeskTicketController(HelpdeskTicketController):
```